--- gnome-devel-docs-3.7.90/platform-demos/Makefile.am.logos-removal	2013-02-04 20:52:08.000000000 +0100
+++ gnome-devel-docs-3.7.90/platform-demos/Makefile.am	2013-02-21 02:01:30.864169035 +0100
@@ -192,7 +192,6 @@
 	media/combobox_multicolumn.png		\
 	media/dialog.png			\
 	media/entry.png				\
-	media/fedora.png			\
 	media/filechooserdialog_menu.png	\
 	media/filechooserdialog_save.png	\
 	media/fontchooserwidget.png		\
@@ -225,7 +224,6 @@
 	media/menubar_choices.png		\
 	media/message-board.ogv			\
 	media/messagedialog.png			\
-	media/opensuse.png			\
 	media/paned.png				\
 	media/photo-wall.png			\
 	media/photo-wall-focused.png		\
@@ -258,7 +256,6 @@
 	media/treeview_simple_liststore.png	\
 	media/treeview_simple_liststore_penguins.png	\
 	media/treeview_treestore.png		\
-	media/ubuntu.png			\
 	media/weatherAppJs.png			\
 	media/window.png			\
 	$(demo_sources)
--- gnome-devel-docs-3.7.90/platform-demos/C/getting-ready.page.logos-removal	2013-02-04 20:52:08.000000000 +0100
+++ gnome-devel-docs-3.7.90/platform-demos/C/getting-ready.page	2013-02-21 02:01:30.864169035 +0100
@@ -36,17 +36,17 @@
 <list>
 	<item><p>
 	<link href="http://live.gnome.org/DeveloperTools/Installation/Ubuntu">
-	<media type="image" mime="image/png" src="media/ubuntu.png">Ubuntu</media>
+	Ubuntu
 	</link></p>
 	</item>
 	<item><p>
 	<link href="http://live.gnome.org/DeveloperTools/Installation/Fedora">
-	<media type="image" mime="image/png" src="media/fedora.png">Fedora</media>
+	Fedora
 	</link></p>
 	</item>
 	<item><p>
 	<link href="http://live.gnome.org/DeveloperTools/Installation/OpenSuSE">
-	<media type="image" mime="image/png" src="media/opensuse.png">OpenSuSE</media>
+	OpenSuSE
 	</link></p>
 	</item>
 	<item><p><link href="http://live.gnome.org/DeveloperTools/Installation">Others</link></p></item>
